es6-persistence/src/main/java/com/netflix/conductor/es6/dao/index/ElasticSearchRestDAOV6.java

```diff
@@ -810,6 +810,95 @@ public CompletableFuture<Void> asyncUpdateWorkflow(
                 () -> updateWorkflow(workflowInstanceId, keys, values), executorService);
     }
 
+    @Override
+    public void removeTask(String workflowId, String taskId) {
+        long startTime = Instant.now().toEpochMilli();
+        String docType = StringUtils.isBlank(docTypeOverride) ? TASK_DOC_TYPE : docTypeOverride;
+
+        SearchResult<String> taskSearchResult =
+                searchTasks(
+                        QueryBuilders.boolQuery()
+                                .must(QueryBuilders.termQuery("workflowId", workflowId))
+                                .must(QueryBuilders.termQuery("taskId", taskId))
+                                .toString(),
+                        "*",
+                        0,
+                        1,
+                        null);
+
+        if (taskSearchResult.getTotalHits() == 0) {
+            LOGGER.error("Task: {} does not belong to workflow: {}", taskId, workflowId);
+        }
```

Comment:
why do we continue execution even after we don't find the task with the given id ?

Reply:
You are correct, I will go ahead and extend the tests for the changes. I will create a separate PR to fix this behaviour in `#removeWorkflow` alongside some other changes that need to be done. Thank you for noticing!